src/application/redux/containers/receive-select-asset.container.ts

```diff
@@ -5,14 +5,23 @@ import { RootReducerState } from '../../../domain/common';
 import ReceiveSelectAssetView, {
   ReceiveSelectAssetProps,
 } from '../../../presentation/wallet/receive/receive-select-asset';
+import { lbtcAssetByNetwork } from '../../utils/network';
+import { selectNetwork } from '../selectors/app.selector';
 import { selectBalances } from '../selectors/balance.selector';
 
 const mapStateToProps = (state: RootReducerState): ReceiveSelectAssetProps => {
   const balances = selectBalances(MainAccountID)(state);
+  const network = selectNetwork(state);
   const getAsset = assetGetterFromIAssets(state.assets);
   return {
     network: state.app.network,
-    assets: Object.keys(balances).map(getAsset),
+    assets: Object.keys(balances).map((assetHash: string) => {
```

Comment:
This selector seems to be reused in several containers. let's create a dedicated select function ?